apps/whale-api/src/module.api/poolpair.service.ts

```diff
@@ -151,7 +149,7 @@ export class PoolPairService {
 
   private async getTokenUSDValue (id: number): Promise<BigNumber | undefined> {
     return await this.cache.get<BigNumber>(`PRICE_FOR_TOKEN_${id}`, async () => {
-      const tokenInfo = await this.tokenMapper.getByTokenId(id)
+      const tokenInfo = await this.deFiDCache.getTokenInfo(`${id}`)
```

Comment:
`nit:` Looking at this method now it does not seem like the `id` is used as a number anymore. So wondering if perhaps instead of converting the incoming id from `number` to `string`. To just set the expected id value to `string` in the method instead? 🤔